deploy/metrics-server-deployment.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: metrics-server
+  namespace: kube-system
+  labels:
+    k8s-app: metrics-server
+spec:
+  selector:
+    matchLabels:
+      k8s-app: metrics-server
+  template:
+    metadata:
+      name: metrics-server
+      labels:
+        k8s-app: metrics-server
+    spec:
+      serviceAccountName: metrics-server
+      containers:
+      - name: metrics-server
+        image: gcr.io/google_containers/metrics-server-amd64:dev
+        imagePullPolicy: Always
+        command:
+        - /metrics-server
+        - --source=kubernetes.summary_api:''
+        - --requestheader-client-ca-file=/var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+        volumeMounts:
+        - name: ssl-certs
+          mountPath: /etc/ssl/certs
```

Comment:
@piosz Test removing it. If it starts failing (which I don't think it will), add it back. Otherwise, remove it, as it adds an unnecessary dependency on the host.

IIUC, the metrics server won't ever hit public domains like `google.com`, will it? 
That's why CA certificates would be required, to verify the authenticity of normal sites...